incubator/vault-operator/values.yaml

```diff
@@ -0,0 +1,71 @@
+## Default values for the image
+name: vault-operator
+replicaCount: 1
+image:
+  repository: quay.io/coreos/vault-operator
+  tag: 0.1.9
+  pullPolicy: Always
+
+## Install Default RBAC roles and bindings
+rbac:
+  create: true
+  apiVersion: v1beta1
+
+## Service account names and whether to create them
+serviceAccount:
+  create: true
+  name: vault-operator-sa
+
+resources:
+  cpu: 100m
+  memory: 128Mi
+## Node labels for vault-operator pod assignment
+## Ref: https://kubernetes.io/docs/user-guide/node-selection/
+nodeSelector: {}
+## additional command arguments go here; will be translated to `--key=value` form
+## e.g., analytics: true
+commandArgs: {}
+## Configurable health checks against the /readyz endpoint that vault-operator exposes
+readinessProbe:
+  enabled: false
+  initialDelaySeconds: 0
+  periodSeconds: 10
+  timeoutSeconds: 1
+  successThreshold: 1
+  failureThreshold: 3
+livenessProbe:
+  enabled: false
+  initialDelaySeconds: 0
+  periodSeconds: 10
+  timeoutSeconds: 1
+  successThreshold: 1
+  failureThreshold: 3
+
+###
+# All of the config variables related to setting up the etcd-operator
+# Disabled by default
+# If you want more information about the variables exposed, please visit:
+#   https://github.com/kubernetes/charts/tree/master/stable/etcd-operator#configuration
+###
+etcd-operator:
+  enabled: false
```

Comment:
Why isn't this enabled by default?

Reply:
I'm not entirely sure - we run `etcd-operator` as a global, so one and done; I think it should be left up to the person using the `vault-operator` to be able to choose if they want 1:1 or 1:n.